AS/src/main/java/org/example/Classify.java

```diff
@@ -0,0 +1,9 @@
+package org.example;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Set;
+
+public interface Classify {
+    public void SetFill(Set<String> set, HashMap<String, String> map, String[] wordsLine);
```

Comment:
Try to rename this function to something more indicative. What does this function do? 
Also, we do not tend to name objects as "set" and "map". Someone who reads this should be able to understand exactly what this map/set hold